tests/NHSD.GPIT.BuyingCatalogue.E2ETests.Framework/Actions/Common/CommonActions.cs

```
@@ -382,13 +384,20 @@ public bool PageLoadedCorrectGetIndex(
             var actionUrl = new Uri("https://www.fake.com/" + absoluteRoute, UriKind.Absolute);
 
```

Isn't this the check to determine if the test fails or passes? Hence it should return false, leading to the test failing gracefully. It should only throw an exception when something unexpected has happened.

---

This is a bit of a hacky way to get extra information out of this check without having to refactor every instance where its used. At the moment it just returns false and doesn't really tell you what went wrong. With the more detailed output I was able to tell that it was still in the process of logging in when it failed

---

Does it not show up in the stack trace?